types/shapediver/appbuilder.ts

```diff
@@ -319,6 +319,17 @@ export interface IAppBuilderContainer {
 	widgets?: IAppBuilderWidget[];
 }
 
+export interface IAppBuilderInstanceDefinition {
+	/** Id of the instance. */
+	sessionId: string;
+	/** Optional name of the instance. This name will be used for the node in the scene graph, e.g. NAME_transformations_0 for the first transformation. */
+	name?: string;
+	/** Parameter set for the instance. If none is provided, the default parameter set is used. */
+	parameterSet?: {[key: string]: string};
```

Comment:
@MajorMeerkatThe3rd let's document here what exactly the keys should be (parameter id, name, displayname)